src/pages/PagePeople/PagePeople.tsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect, useState } from 'react';
+import { Link, useNavigate, useParams } from 'react-router-dom';
+import { ApiProduct } from '../../api/Api';
+import { Grogu } from '../../components/Grogu/Grogu';
+import { IdataCardOnePeople, IdataPeople } from '../../type/interfaces';
+import style from './_pagePeople.module.scss';
+import DeleteLogo from '/assets/images/delete.png';
+
+export function PagePeople(): JSX.Element {
+  const [groguSpinner, setGroguSpinner] = useState<boolean>(false);
```

Comment:
boolean variables are usually named like a question? isLoading, isPending or hasValidSymbols and so on